scroll-animations-1/Overview.bs

```diff
@@ -347,6 +347,57 @@ spec: cssom-view-1; type: dfn;
 	In the Web Animations API,
 	they can be represented anonymously by a {{ViewTimeline}} object.
 
+## View Progress Timeline Ranges ## {#view-timelines-ranges}
+
+	[=View progress timelines=] define the following [=named timeline ranges=]:
+
+	<dl dfn-for="animation-timeline-range" dfn-type="value">
+		<dt><dfn>cover</dfn>
+		<dd>
+			Represents the full range of the [=view progress timeline=]:
+			* 0% progress represents the position at which
+				the [=end=] [=border edge=] of the element’s [=principal box=]
+				coincides with the [=start=] edge of its [=view progress visibility range=].
+			* 100% progress represents the position at which
+				the [=start=] [=border edge=] of the element’s [=principal box=]
+				coincides with the [=end=] edge of its [=view progress visibility range=].
+
+		<dt><dfn>contain</dfn>
+		<dd>
+			Represents the range during which the [=principal box=]
+			is either fully contained by, or fully covers,
+			its [=view progress visibility range=] within the [=scrollport=].
+			* 0% progress represents the earlier position at which:
```

Comment:
Similar remark to the one above about the “View Progress Visibility Range” (vpvr) – it depends on how you look at it when naming the edges.

Maybe split this up differently so it talks about subjects smaller/larger then the possible area?

- For subjects whose size is smaller than the vpvr:
  - 0% progress = the end border edge of the element’s principal box coinciding with the end edge of the vpvr
  - 100% progress = the start border edge of the element’s principal box coinciding with the start edge of the vpvr
- For subjects whose size is larger than the vpvr:
  - 0% progress = the start border edge of the element’s principal box coinciding with the start edge of the 
  - 100% progress = the end border edge of the element’s principal box coinciding with the end edge of the

Reply:
> Similar remark to the one above about the “View Progress Visibility Range” (vpvr) – it depends on how you look at it when naming the edges.

Edges are named according to the writing mode, this is already defined at the bottom of the section.

> Maybe split this up differently so it talks about subjects smaller/larger then the possible area?

Sure, could do that. Maybe in a follow-up commit?